src/App.tsx

```diff
@@ -1,14 +1,66 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo>();
+  const [isTodoModalOpen, setIsTodoModalOpen] = useState(false);
+  const [selected, setSelected] = useState('all');
+  const [searched, setSearched] = useState('');
+  const isActiveModal = selectedTodo && isTodoModalOpen;
+  const [activeTodo, setActiveTodo] = useState(selectedTodo?.id || -1);
+
+  useEffect(() => {
+    getTodos()
+      .then((todosFromServer) => setTodos(todosFromServer));
```

Comment:
would be better to use `async/await` syntax
```suggestion
      .then(setTodos);
```